service/src/main/java/bio/terra/workspace/service/policy/TpsApiDispatch.java

```diff
@@ -194,16 +197,20 @@ public List<String> listValidRegions(UUID workspaceId, String platform) {
     return new ArrayList<>();
   }
 
-  public TpsLocation getLocationInfo(String platform, String location) {
+  public TpsLocation getLocationInfo(CloudPlatform platform, String location) {
     features.tpsEnabledCheck();
     TpsApi tpsApi = policyApi();
     try {
-      return tpsApi.getLocationInfo(platform, location);
+      return tpsApi.getLocationInfo(convertCloudPlatformToTpsCloudPlatform(platform), location);
     } catch (ApiException e) {
       throw convertApiException(e);
     }
   }
 
+  private static String convertCloudPlatformToTpsCloudPlatform(CloudPlatform platform) {
+    return platform.toSql().toLowerCase(Locale.ROOT);
```

Comment:
You assume here that the SQL string is identical to the TPS string.
I think those should be independent.

Suggest in CloudPlatform add
```
private final tpsString;
```
Fill the new field in on each enum. (BTW, looks like AZURE needs to have the ApiCloudPlatform set; it shouldn't be `null`!)

Add a getter, maybe like:
```
public String getTpsString() {
 if (tpsString == null) { throw InternalLogicError(); }
 return tpsString;
}
```